packages/edgeidentity/README.md

```diff
@@ -108,3 +108,8 @@ AEPIdentity.extensionVersion().then(version => console.log("AdobeExperienceSDK:
 ```javascript
 AEPIdentity.getExperienceCloudId().then(experienceCloudId => console.log("AdobeExperienceSDK: Experience Cloud Id = " + experienceCloudId));
 ```
+#### Get Experience Cloud ID with Promise rejection check:
+```javascript
+AEPIdentity.getExperienceCloudId().then(experienceCloudId => {console.log("AdobeExperienceSDK: Experience Cloud Id = " + experienceCloudId);}, error => {console.error(error);});
+```
```

Comment:
Can we keep only one example in here like follows:
#### Get Experience Cloud ID:
```javascript
AEPIdentity.getExperienceCloudId().then(experienceCloudId => {console.log("AdobeExperienceSDK: Experience Cloud Id = " + experienceCloudId);}, error => {console.error(error);});
```

Reply:
We should also update the documentation in AEPIdentity.js 
`@return promise method which will be invoked once the Experience Cloud ID is available or rejected if an unexpected error occurred or the request timed out.`

Reply:
updated